src/Countries/India.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Spatie\Holidays\Countries;
+
+use Carbon\CarbonImmutable;
+
+class India extends Country
+{
+    public function countryCode(): string
+    {
+        return 'in';
+    }
+
+    protected function allHolidays(int $year): array
+    {
+        return [
+            'Republic Day' => '01-26',
+            'Holi' => '03-25',
+            'Good Friday' => '03-29',
+            'Id-ul-Fitr' => '04-11',
+            'Ram Navmi' => '04-17',
```

Comment:
Diwali, Dussehra, Ram Navmi, Republic are not match the English day every year they follow tithi